### PR TITLE
updated line 28

### PR DESCRIPTION
**Pull request by Shakir101**
just a beginner making trying out pull requests

### CODE REVIEW

**Review by mahyuddin1359 — COMMENTED** [review]
good

File: gcm-sender-async/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
             <groupId>com.squareup.okhttp</groupId>
             <artifactId>mockwebserver</artifactId>
             <version>2.1.0</version>
-            <scope>test</scope>
+            <scope>tests</scope>
```
Comment (mpires) [inline gcm-sender-async/pom.xml:28]: The proper maven scope here is actually test

**Comment by maks1m** [comment]
> just a beginner making trying out pull requests

In this case I would recommend you to read https://chris.beams.io/posts/git-commit/

**Review by ohbus — CHANGES_REQUESTED** [review]
Please update yourself in these matters.

**Review by JJS777 — APPROVED** [review]

**Review by JJS777 — APPROVED** [review]

**Review by JJS777 — APPROVED** [review]

File: gcm-sender-async/pom.xml (hunk 1 above)
Comment (Astrodevil) [inline gcm-sender-async/pom.xml:28]: yeah here is the (test) please remove s from it. the scope is test not tests.

**Review by Astrodevil — APPROVED** [review]

**Pull request closed without merging**